src/debugpy/server/cli.py

```diff
@@ -325,6 +327,8 @@ def start_debugging(argv_0):
             raise AssertionError(repr(options.mode))
 
         if options.wait_for_client:
+            if options.log_on_listening:
+                print(f"Listening for debugger connection on {options.address[0]}:{options.address[1]}")
```

Comment:
If you did so you could put a `log.debug` here or something with a higher trace level so that it would only show up when the log level was high enough.

Reply:
Stupid question, but is there a way to set log level? The only logging options are `log-to-stderr` and `log-to=[file]`, neither of which set level.

Reply:
log-to file seems to set the level to debug. I don't see a way to set the pydevd log level from debugpy besides that though. log-to-stderr does the same thing.

Reply:
Could you use log-to file and have the problem matcher read the file instead?

Reply:
No, [problemMatcher](https://code.visualstudio.com/docs/editor/tasks#_defining-a-problem-matcher) doesn't support reading files. And that sounds like a more complex solution compared to the relatively reasonable/straightforward approach here.

Reply:
`--log-to-stderr` technically works, but prints ~260 lines of excess logs. IMO this log should be INFO or WARN, so how can I set stdout log level to info?